src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+interface PaginationProps {
+  total: string[];
```

Comment:
The `total` prop should be of type `number`, not `string[]`. It should represent the total number of items, not the items themselves. Update this to `total: number;`.